CMAKE preprocess_lv_conf_internal.py  fails on Windows

### LVGL version

v9.3

### Platform

Windows 10 x64

### What happened?

When `-DLV_BUILD_SET_CONFIG_OPTS=1` was specified on the command line, CMAKE launches `preprocess_lv_conf_internal.py`, and that script uses `which` to get the path to `pcpp` executable, which on windows, outputs something like `/c/Python/Python312/Scripts/pcpp` which is not a valid Windows path.  It works under `bash`, but not from a Python script.




### How to reproduce?

```
cmake -B build -DLV_BUILD_CONF_PATH="path/to/lv_conf.h" -DLV_BUILD_SET_CONFIG_OPTS=1
```
on Windows.

Output:
```
-- Selecting Windows SDK version 10.0.22621.0 to target Windows 10.0.19045.
-- Found Python: C:/Python/Python312/python.exe (found version "3.12.6") found components: Interpreter
-- E:/Dev/Clients/WGA/lvgl/vwheeler63/build/vs/lv_conf.h
-- Using configuration: E:/Dev/Clients/WGA/lvgl/vwheeler63/build/vs/lv_conf.h
Found PCPP: /c/Python/Python312/Scripts/pcpp
Traceback (most recent call last):
  File "E:\Dev\Clients\WGA\lvgl\vwheeler63\lvgl\scripts\preprocess_lv_conf_internal.py", line 192, in <module>
    main()
  File "E:\Dev\Clients\WGA\lvgl\vwheeler63\lvgl\scripts\preprocess_lv_conf_internal.py", line 172, in main
    preprocess_file(pcpp_exe, args.input, args.tmp_file, args.output, args.include, args.defs)
  File "E:\Dev\Clients\WGA\lvgl\vwheeler63\lvgl\scripts\preprocess_lv_conf_internal.py", line 55, in preprocess_file
    subprocess.run(pcpp_command, check=True)
  File "C:\Python\Python312\Lib\subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python\Python312\Lib\subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Python\Python312\Lib\subprocess.py", line 1538, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified
CMake Error at env_support/cmake/os_desktop.cmake:170 (message):
  preprocess_lv_conf_internal.py failed with return code: 1
Call Stack (most recent call first):
  CMakeLists.txt:23 (include)


-- Configuring incomplete, errors occurred!
```

all because `which pcpp` outputs:  `/c/Python/Python312/Scripts/pcpp`.

--- a/scripts/preprocess_lv_conf_internal.py
+++ b/scripts/preprocess_lv_conf_internal.py
@@ -14,6 +14,7 @@ import subprocess
 import os
 import argparse
 import re
+import shutil
 import importlib.util
 
 
@@ -158,9 +159,10 @@ def main():
     # Check if PCPP is already present on the system
     # if it's not - create a python venv inside the workfolder directory
     # and install it there
-    res = subprocess.run(["which", "pcpp"], capture_output=True)
-    if res.returncode == 0:
-        pcpp_exe = res.stdout.decode().replace("\n", "")
+    pcpp_path = shutil.which("pcpp")
+
+    if len(pcpp_path) > 0:
+        pcpp_exe = pcpp_path
         print(f"Found PCPP: {pcpp_exe}")
     else:
         print("Failed to locate pcpp - installing it")

--- a/scripts/preprocess_lv_conf_internal.py
+++ b/scripts/preprocess_lv_conf_internal.py
@@ -161,7 +161,7 @@ def main():
     # and install it there
     pcpp_path = shutil.which("pcpp")
 
-    if len(pcpp_path) > 0:
+    if pcpp_path:
         pcpp_exe = pcpp_path
         print(f"Found PCPP: {pcpp_exe}")
     else:

